DOCUMENTATION.md

```diff
@@ -185,13 +185,17 @@ email_reply_to_id='8e222534-7f05-4972-86e3-17c5d9f894e2' # optional UUID string
 
 If you omit this argument, the client will set your default email reply-to address for the notification.
 
-### Upload a document
+### Send a document by email
 
-You can upload a document to link to from an email notification.
+You can send files without the need for email attachments. First upload a document, then send recipients a link to download it.
 
-The document must be a PDF file smaller than 2MB.
+To enable this function for your service [contact the GOV.UK Notify team](https://www.notifications.service.gov.uk/support).
```

Comment:
Perhaps "[Contact the GOV.UK Notify team] to enable this function for your service." Puts the action up front.